src/com/projectkorra/projectkorra/ability/PlantAbility.java

```diff
@@ -20,4 +23,9 @@ public Element getElement() {
 		return Element.PLANT;
 	}
 
+	// Because Plantbending deserves particles too!
+	public void playPlantbendingParticles(final Location loc, final int amount, final double xOffset, final double yOffset, final double zOffset) {
+		loc.getWorld().spawnParticle(Particle.BLOCK_CRACK, loc.clone().add(0.5, 0, 0.5), amount, xOffset, yOffset, zOffset, Material.OAK_LEAVES.createBlockData());
```

Comment:
We should add a config option for this rather than making a default.

Reply:
I think it's fine. Nothing uses it as of now, so having it for future abilities to use in case they want to is fine imo

Reply:
Not sure what you mean by default. Which value? The OAK_LEAVES ?